openmc/tallies.py

```diff
@@ -1468,32 +1518,23 @@ def _outer_product(self, other, binary_op):
             new_name = '({0} {1} {2})'.format(self.name, binary_op, other.name)
             new_tally.name = new_name
 
+        # Query the mean and std dev so the tally data is read in from file
+        # if it has not already been read in.
+        self.mean
```

Comment:
I wonder if these can be comma separate on the same line? Perhaps this is not PEP8-compliant. Not a big deal, I just wonder if these lines could be compressed.